get_webstat_metrics.py

```diff
@@ -162,10 +162,10 @@ async def main():
 
         if node_mainnet_count <= 3:
             output['status']['mainnet'] = "Severe"
-            output['statusmsgs']['mainnet'].append(f"{node_mainnet_count} / 5 nodes is available")
+            output['statusmsgs']['mainnet'].append(f"{node_mainnet_count} out of 5 nodes {'is' if node_mainnet_count == 1 else 'are'} available")
```

Comment:
i do not want believe that we ever see "0" but if yes, i think it is also "is"

Reply:
zero it's also `are`

https://english.stackexchange.com/a/38310